projects/leds/src/main.c

```diff
@@ -4,9 +4,10 @@
 #include <stddef.h>
 #include <stdint.h>
```

Comment:
Make sure to remove any extra files that are not part of the PR. We normally want to clean our pull request clean and focused on one ticket without extraneous files. In this case we would want to focus on only the FW 102 hwk